pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -43,8 +43,9 @@ type TargetConfigController struct {
 
 	operatorClient v1helpers.StaticPodOperatorClient
 
-	kubeClient      kubernetes.Interface
-	configMapLister corev1listers.ConfigMapLister
+	kubeClient          kubernetes.Interface
+	configMapLister     corev1listers.ConfigMapLister
+	infrastuctureLister configv1.InfrastructureLister
```

Comment:
nit: This appears to be unused? 

My recent experience with openshift/kubernetes suggests that it might be time for build-machinery-go to support a `make vet` check so that static analysis can catch these kinds of gotchas.

Reply:
It is ran inside verify but `go vet` doesn't catch this. Technically it isn't an error because someone using this as a library, could rightfully use it. This isn't the case here but I am not sure computer can know, but thanks for catching this.